cmd/agent/gui/views/private/js/javascript.js

```diff
@@ -157,9 +157,9 @@ function loadStatus(page) {
       // Get the trace-agent status
       sendMessage("agent/getConfig/apm_config.receiver_port", "", "GET",
           function(data, status, xhr) {
-              var apmPort = data["apm_config.receiver_port"];
+              var apmPort = data["apm_config.debug_server.port"];
```

Comment:
What's the reason behind `debug_server.port` instead of just `debug.port`. Is there a client that we need to be able to differentiate?

Reply:
No there is no debug client afaik. I like `debug.port`, it's shorter and descriptive enough! Thanks for the suggestion!